### PR TITLE
Set up Gemini repo skeleton structure

### PR DESCRIPTION
**Pull request by HamraKhan**
- set up node express with react
- configured vs code to run app locally using one terminal
- added necessary testing libraries/packages

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,45 @@
+{
+  "name": "gemini-weather",
+  "version": "1.0.0",
+  "description": "gemini weather app",
+  "main": "server.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "concurrently \"nodemon server.js\" \"cd ./client && npm start\"",
```
Comment (navid230) [inline package.json]: ensure that you can use it concurrently in a production environment. also check if nodemon is suitable for production running of the server side.

Comment (navid230) [inline package.json]: create a dev script and move the start to dev.

File: package.json [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    "url": "https://github.com/HamraKhan/gemini-weather/issues"
+  },
+  "homepage": "https://github.com/HamraKhan/gemini-weather#readme",
+  "dependencies": {
+    "body-parser": "^1.20.0",
+    "dotenv": "^16.0.2",
+    "express": "^4.18.1",
+    "lodash": "^4.17.21",
+    "moment": "^2.29.4",
+    "nodemon": "^2.0.19",
```
Comment (navid230) [inline package.json]: move nodemon to devDependencies.

**Review by navid230 — CHANGES_REQUESTED** [review]
please make the appropriate changes, thanks.

**Comment by navid230** [comment]
as discussed, here is a sample implementation of the cluster module. it'll be great if we can use this in the node server
https://blog.appsignal.com/2021/02/03/improving-node-application-performance-with-clustering.html

File: package.json (hunk 1 above)
Comment (HamraKhan) [reply package.json]: done :)

Comment (HamraKhan) [reply package.json]: done :)

File: package.json (hunk 2 above)
Comment (HamraKhan) [reply package.json]: done :)

**Review by navid230 — APPROVED** [review]

**Pull request merged**